samri/optimization/registration.py

```diff
@@ -145,10 +153,73 @@ def structural_rigid_affine(template="/Users/marksm/GitHub/mriPipeline/ants_test
 	struct_registration.inputs.args = '--float'
 	struct_registration.inputs.num_threads = 6
 
-	struct_registration.inputs.moving_image = n4_res.outputs.output_image
+	struct_registration.inputs.moving_image = flt_res.outputs.out_file
+	# struct_registration.inputs.output_warped_image = 'ss_{}_ofM{}.nii.gz'.format(participant,i)
+	struct_registration.inputs.output_warped_image = output_image
+	res = struct_registration.run()
+
+def structural_flirt_affine(template="/Users/marksm/GitHub/mriPipeline/ants_test/template4.nii.gz",
```

Comment:
and this function here uses FLIRT for all things linear? Again, maybe a more descriptive name and/or a docstring?